swift/ios-shoppe-demo/ServiceManagers/APIService.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  APIService.swift
```

Comment:
@HaroldFS this looks like another duplicated file. Would you please verify:

![image](https://user-images.githubusercontent.com/45576380/76553599-b38e9b80-646a-11ea-8f18-e1deb64216e3.png)